### DISCUSSION TITLE
My lazer plays dont submit.

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22224] [Q&A] [answered]
When in multiplayer lobbies I saw that some people actually have a rank thats changing from their lazer plays, while mine continuously decays even though i play alot of lazer. So i set 2 scores just for demos, and none of them show up in my best performances, even though they should be in there. how can that happen?

![image](https://user-images.githubusercontent.com/87947227/212561963-f42a2dd7-79a3-46d9-8e58-ebb888eec908.png)
![image](https://user-images.githubusercontent.com/87947227/212562014-7d211de4-23f2-4c44-8864-ca9b67317d09.png)
![image](https://user-images.githubusercontent.com/87947227/212562035-a408adfd-a5fa-4060-a058-25838d758791.png)
![image](https://user-images.githubusercontent.com/87947227/212562060-a116f7a8-54ef-4339-bb1a-fdb518ab03dc.png)

**Comment by mk56-spn** [comment]
Are they showing up in recent plays?

**Reply by Inconsist** [reply]
no

**Comment by bdach** [comment, marked as answer]
I see them here: https://lazer.ppy.sh/users/17637346

![1673820865](https://user-images.githubusercontent.com/20418176/212569979-c584ccbe-8e5c-45f7-92d3-d7d7c981b472.png)

The pp values do not match, which is a known issue and tracked as https://github.com/ppy/osu/issues/21815.

**Reply by Inconsist** [reply]
Ah when he asked me about recent plays i looked it up in the game client, they didnt show there but the pp mismatch explains the rest.
